include/eigenvalue_hafnian.hpp

```diff
@@ -171,14 +171,14 @@ inline T do_chunk_loops(std::vector<T> &mat, std::vector<T> &C, std::vector<T> &
         comb[0] = 1.0;
 
         for (i = 1; i <= n / 2; i++) {
-            factor = traces[i - 1] / (2.0 * i);
+            factor = traces[i - 1] / static_cast<T> (2.0 * i);
```

Comment:
Has the file been run through astyle? I just noticed that this casting looks like `static_cast<t> ()` but the one above has different formatting `static_cast<T>()`.